src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier supplier = new ColorSupplier();
+        int number = random.nextInt(100);
+        String color = String.valueOf(supplier.getRandomColor());
```

Comment:
Use the name() method instead of String.valueOf() for getting the String representation of enum constants to avoid issues if the toString() method is overridden in the enum class.